pkg/controller/mizar/mizar-arktos-network-controller.go

```diff
@@ -190,9 +191,18 @@ func (c *MizarArktosNetworkController) processNetworkCreation(network *v1.Networ
 	code := response.Code
 	context := response.Message
 
-	if code != CodeType_OK {
-		return errors.New("Fail to update VPC with Arktos Network name on the Mizar cluster")
+	switch code {
+	case CodeType_OK:
+		klog.Infof("Mizar handled arktos network creation successfully: %s", key)
```

Comment:
log: only create events will hit the code in line 196

Reply:
During arktos network creation, on mizar side, we need associate akrtos network obj name with VPC ID. 
I agree the log is not very clear, I will update it